packages/components/src/DatePicker/DatePicker.tsx

```diff
@@ -123,11 +135,13 @@ export function DatePicker({
         onCalendarOpen={handleCalendarOpen}
         onCalendarClose={handleCalendarClose}
         dateFormat={["P", "PP", "PPP", "MMM dd yyyy", "MMMM dd yyyy"]}
+        highlightDates={highlightDates}
+        onMonthChange={onMonthChange}
       />
     </div>
   );
 
-  function handleChange(value: Date) {
+  function handleChange(value: Date /* , event: React.SyntheticEvent */) {
```

Comment:
any reason why you kept that comment?

Reply:
So that the next person to come along and think "oh, this isn't doing anything I can remove it" might realize that there's that small difference in the call signature. That said, maybe it doesn't really communicate _why_ the difference matters 🤔

Reply:
yeah, i'd leave a description around why

Reply:
Added a description for why `handleChange` is necessary here.